ts-packages/types/src/types/rust/RewardEstimate.ts

```diff
@@ -1,6 +1,6 @@
-export type RewardEstimate = {
+export interface RewardEstimate {
```

Comment:
Yeah, I dont think you want to be changing it here, because whenever somebody regenerates the types it will get overwritten

Reply:
yes the change here is auto generation overriding previous handwritten type